CHANGELOG.asciidoc

```diff
@@ -35,6 +35,9 @@ This release also includes changes from <<release-3-3-8, 3.3.8>>.
 * Bump to Netty 4.1.36.
 * Bump to Groovy 2.5.7.
 
+==== Bugs
```

Comment:
There is no need to add this manually - we add those on release. You should however add a manual changelog entry above that describes your change.

Reply:
Ok, I have removed changes in  CHANGELOG.asciidoc

Reply:
python test fail, and i thinks it's travis-ci problem also